### DISCUSSION TITLE
Ursina Lighting Question

### DISCUSSION THREAD

**Discussion by Tungara** [pokepetter/ursina#415] [General]
I am making a simple simulation of the solar system in Ursina and I am trying to implement a shader. I haven't been able to set the sun to emanate light. I have already tried using point light and setting the location in the sun but this makes the sun turn black. I tried to disable the sun's collision box and set the shader to None but it didn't seem to work. If anybody can provide a solution it would be greatly appreciated.

Thanks!

**Comment by pokepetter** [comment]
What is the shader you're trying to implement?

**Reply by Tungara** [reply]
I am new to Ursina so I am not sure if there is a better shader but I am currently using the lit_with_shadows_shader.

**Comment by pokepetter** [comment]
Also, the title says "Ursina Lighting Question", but what exactly is the question?

**Reply by Tungara** [reply]
My question is if there is a way of stopping an entity from being affected by a light source or shader. Another alternative would be making the entity a light source instead of placing one inside of it but I am not sure if this is possible.

**Reply by pokepetter** [reply]
Yeah, try the unlit shader or setting .unlit=True on the entity

**Reply by Tungara** [reply]
Thank you so much for your help! Sorry if my question was a little basic.
